src/components/ui/NumericText/index.js

```diff
@@ -1,28 +1,52 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import { injectIntl } from 'react-intl';
+import BigNumber from 'bignumber.js';
 import SelectableText from '../SelectableText';
 
-const NumericText = (props) => {
-  const { value, type, intl } = props;
+export const NumericText = (props) => {
+  const { value, type, maxDecimalPlaces, intl } = props;
   const { formatNumber } = intl;
-  const options = {
-    maximumFractionDigits: 6,
-  };
+
   if (type === 'currency') {
```

Comment:
Is `type==='currency'` only for fiat currencies? 

The reason I'm asking ia because maxDecimalPlaces is set to 18 a lot of places which looks to be currencies as well, but maybe these are non-fiat currencies?

If so, would it be an option to define another `type` which is basically "crypto currency' that defaults to 18 max decimal places? (To avoid having to set maxDecimalPlaces to 18 so many places)

Reply:
Good spot. Yes, it is only for the USD fiat currency. Perhaps this type value needs to be renamed to `fiat` instead.